kratos/processes/find_intersected_geometrical_objects_process.cpp

```diff
@@ -28,8 +28,8 @@ template<class TEntity>
 FindIntersectedGeometricalObjectsProcess<TEntity>::FindIntersectedGeometricalObjectsProcess(
     ModelPart& rPart1,
     ModelPart& rPart2
-    ) : mrModelPartIntersected(rPart1),
```

Comment:
Part 1 is supposed to be the volume part, which is the intersected one. This was correct. What was wrong was the new methods you created.

Reply:
The methods I created followed exactly was was done until now

Reply:
It is not a matter of search and replace, the implementation of the new specialized methods was wrong.

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/4e433cb22a51ecc2a8ea444adbeb4b0bedda31c6/kratos/processes/find_intersected_geometrical_objects_process.cpp#L41, in that case waht was wrong was  the name of the method

Reply:
Fixed